### DISCUSSION TITLE
现在有多少种食物？How many types of food are there?

### DISCUSSION THREAD

**Discussion by Silvigarabis** [Rad586/Forget-Me-Not#78] [General]
整合包的一个设定是如果死亡就会损失血量上限，而吃到新的食物可以增加血量上限，除此之外似乎便没有改变血量上限的方法了。那么，食物的种类数量就变得重要起来了，到底有多少种食物？

**Comment by Rad586** [comment]
200+
血量上限每次只减少 0.2，死10次掉1心，50次5心，不是很大的数字。
It only decreases your health by 0.2, so you lose 1 heart for 10 deaths. Not something like that serious.

**Reply by Silvigarabis** [reply]
> 200+ 血量上限每次只减少 0.2，死10次掉1心，50次5心，不是很大的数字。 It only decreases your health by 0.2, so you lose 1 heart for 10 deaths. Not something like that serious.

我觉得人们最担心的事情是，目前增加血量上限的机会是“可以被用尽的”，但是死亡发生的次数是“可以无限次发生的”，所以人们就会对这种事情感到担忧，毕竟人们是感性的，只看数字并不能让这种感觉消失。我觉得这在相当程度上增加了整合包的负面反馈

**Reply by Rad586** [reply]
我能理解一些玩家的焦虑感。但是，我认为像奇异博士一样，采用反复重生的消耗玩法，比一些微乎其微的死亡惩罚来得更加消极。这是刻意为之，且经过深思熟虑的。
I do understand some players might be frustrated with this feature. But for me, it's more than negative to respawn continuously without any consequences, like Dr. Strange. This feature is intended and considered.
